src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+(() => {
+  const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+  if (!dashboard) return;
+
+  const visiblePeople = people.map((person) => ({
+    name: person.name,
+    sex: person.sex,
+    born: person.born,
+    died: person.died,
+    age: person.died - person.born,
+    century: Math.ceil(person.died / 100),
+  }));
+
+  for (let i = 0; i < visiblePeople.length; ++i) {
+    const row = dashboard.insertRow();
+
+    for (const c of Object.keys(visiblePeople[i])) {
+      row.insertCell().textContent = visiblePeople[i][c];
+    }
+  }
```

Comment:
This loop could be more readable if you used more meaningful variable names. For instance, 'c' could be renamed to 'column' or 'property'. This would make it immediately clear what the variable represents.